### PR TITLE
Align X in CQL bad query

### LINKED ISSUE #296: CQL X alignment isn't always correct
On Chrome 39.0.2171.65, OS X:

image

When I set `.query-status.bad:before ` to `left: 0px`, it centers properly

### PR DESCRIPTION
**Pull request by ihodes**
Can't repro screenshot (could be a font issue?)

fixes #296

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/cycledash/437)

### CODE REVIEW

**Comment by danvk** [comment]
add a screenshot? LGTM otherwise.

**Comment by tavinathanson** [comment]
Oops, missed this one, :+1:

**Comment by ihodes** [comment]
@danvk There is a screenshot—not sure what you mean?

**Comment by danvk** [comment]
Shouldn't this PR show a change in the screenshot then?

**Comment by ihodes** [comment]
Do you mean the dpxdt screenshots? None of those show bad queries, so this wouldn't change any of them.

**Comment by danvk** [comment]
Yeah, I'm talking about dpxdt. That's why I suggested adding a screenshot -- it's a UI state that's currently untracked.

**Comment by ihodes** [comment]
Ah, I thought you were referring to a screenshot, not a test! 

I'm a little averse to adding a screenshot test for a little thing like this given that each tests adds about 5+ seconds to my tests (and probably longer for @tavinathanson). Is the marginal benefit we gain from tracking states like these worth the repeated pain?

**Comment by danvk** [comment]
We should solve that by figuring out how to make the tests run faster, not by avoiding writing tests!

**Comment by ihodes** [comment]
I agree 100%.

**Comment by ihodes** [comment]
Finally did this; good to go? The test looks wonky due to the font we're using… this is something we should probably work out. At some pt, these should be icons anyway.

**Comment by tavinathanson** [comment]
@isaac if the bug is the alignment of the X, and the alignment looks wonky in the test, that sounds to me like an unhelpful test?

File: tests/pdifftests/test.yaml [hunk 1]
```diff
@@ -78,6 +78,18 @@ tests:
...
         }, 250);
 
+  - name: examine-bad-query
+    description: Examine page showing a poorly formed query
+    url: http://localhost:5001/runs/1/examine
+    config:
+      <<: *std-config
+      injectJs: |
+        setTimeout(function() {
+          q = $('.query-status').first();
```
Comment (danvk) [inline tests/pdifftests/test.yaml]: FYI the `.first()` isn't necessary (but it doesn't hurt when there's only one matched element).

**Comment by danvk** [comment]
LG, thanks for adding the screenshot!

**Comment by ihodes** [comment]
@tavinathanson yeah, there's just not a way to make it work otherwise, unless we add icons. Problem is with PhantomJS + our custom font.

**Pull request merged**